src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -3665,15 +3680,8 @@ boolean group(Context context, DeclarationList declList) throws ParserException
                 inheritedConstructors += d.text;
             }
         }
-        String constructorName = originalName;
```

Comment:
This failed for nested templates, for example in Pytorch with `c10::static_cast_with_inter_type<c10::complex<c10::Half>,c10::complex<double> >`